Code/GraphMol/MolPickler.h

```diff
@@ -55,7 +55,8 @@ typedef enum {
   PrivateProps = BOOST_BINARY(10000),
   ComputedProps = BOOST_BINARY(100000),
   AllProps =
-      0x7FFFFFFF,  // all data pickled (only 31 bit flags in case enum==int)
+      0x0000FFFF,  // all data pickled (only 31 bit flags in case enum==int)
+  ConfDouble = 0x0001FFFF // save 3D confs as double
```

Comment:
It's not just 3D confs.
How about `// save coordinates in conformers as doubles`